### PR TITLE
Add some argument parsing options for debugging

### PR DESCRIPTION
**Pull request by PaulSonOfLars**

### CODE REVIEW

File: WLC/image_processing/character.py [hunk 1]
```diff
@@ -9,17 +9,20 @@
...
+                         show_pic, show_line, show_word, show_char)
         self.ocr = OCR()
 
+        if self.show_char:
+            cv2.imshow("Character", image)
+            cv2.waitKey(0)
+
     def get_code(self):
-        # TODO: plugin to tensorflow
+        # TODO: plugin to Tensorflow/Keras
```
Comment (mishrabhinav) [inline WLC/image_processing/character.py]: We have already plugged Keras in this function. I think we should remove this before we this MR is accepted.

Comment (mishrabhinav) [inline WLC/image_processing/character.py]: We have already plugged Keras in this function. I think we should remove this before we this MR is accepted.

File: WLC/image_processing/character.py [hunk 2]
```diff
@@ -45,4 +48,9 @@ def transform_to_standard(self):
         # ideally, we'd want to strip from the top and pad the sides to make it a square. TODO as future optimisation
         res = cv2.resize(img, (STD_IMAGE_SIZE, STD_IMAGE_SIZE))
 
+        if self.show_char:
+            # cv2.imshow("Original Character", img)
```
Comment (mishrabhinav) [inline WLC/image_processing/character.py:48]: Is this commented out `imshow` going to be used?

File: WLC/image_processing/extended_image.py [hunk 3]
```diff
@@ -2,12 +2,27 @@
...
-    def __init__(self, image, x, y, w, h):
+    def __init__(self, image, x_axis, y_axis, width, height, extended_image=None,
+                 show_pic=False, show_line=False, show_word=False, show_char=False):
         self._image = image
-        self._x = x
-        self._y = y
-        self._w = w
-        self._h = h
+        self._x_axis = x_axis
+        self._y_axis = y_axis
```
Comment (mishrabhinav) [inline WLC/image_processing/extended_image.py:8]: Just out of curiosity, is there a specific reason why we moved from `_x` to `_x_axis`?

File: WLC/image_processing/character.py [hunk 4]
```diff
@@ -9,17 +9,20 @@
 
 
 class Character(ExtendedImage):
-    def __init__(self, image, x, y, w, h):
-        super().__init__(image, x, y, w, h)
+    def __init__(self, image, x_axis, y_axis, width, height, extended_image=None,
+                 show_pic=False, show_line=False, show_word=False, show_char=False):
```
Comment (mishrabhinav) [inline WLC/image_processing/character.py]: We should enclose these booleans in a class and pass that around instead.

Comment (PaulSonOfLars) [reply WLC/image_processing/character.py]: Doing that now, had planned to do at later date but no point delaying it

File: WLC/image_processing/character.py (hunk 1 above)
Comment (PaulSonOfLars) [reply WLC/image_processing/character.py]: My bad!

File: WLC/image_processing/extended_image.py (hunk 3 above)
Comment (PaulSonOfLars) [reply WLC/image_processing/extended_image.py:8]: single char variables aren't pythonic, this makes pylint happier

**Review by martinzlocha — APPROVED** [review]

**Pull request merged**